CMakeLists.txt

```diff
@@ -26,13 +26,86 @@ option(ZIG_TEST_COVERAGE "Build Zig with test coverage instrumentation" OFF)
 # git log -p -- deps/lld
 option(ZIG_FORCE_EXTERNAL_LLD "If your system has the LLD patches use it instead of the embedded LLD" OFF)
 
+find_package(LLVM REQUIRED CONFIG)
+include_directories(${LLVM_INCLUDE_DIRS})
 
-find_package(llvm)
-include_directories(${LLVM_INCLUDE_DIR})
-link_directories(${LLVM_LIBDIRS})
+# TODO: this currently doesn't work, it currently defines UNICODE but zig
+#       uses MBCS
```

Comment:
I read the docs about MBCS but it's not quite clear to me. What is zig doing to choose MBCS? Would it be better to choose UNICODE?